account_vat_period_end_statement/tests/test_vat_statement.py

```diff
@@ -398,3 +399,56 @@ def test_different_previous_vat_statements(self):
             other_statement.previous_credit_vat_amount,
             -other_last_year_statement.authority_vat_amount,
         )
+
+    def _add_period(self, statement, date_range):
```

Comment:
sarebbe possibile riutilizzare questo metodo in https://github.com/OCA/l10n-italy/blob/dc67f116c1550e6aa7f592c459cf1be7083414b7/account_vat_period_end_statement/tests/test_vat_statement.py#L305-L313?

Reply:
Grazie della revisione!
Sì certo ho modificato per riutilizzare il metodo che esisteva già, puoi vedere se ora va meglio?

Ho anche dovuto fare un rebase altrimenti c'erano conflitti con `16.0`.